src/components/tests/site-content-tabs.test.tsx

```diff
@@ -61,6 +62,7 @@ describe( 'SiteContentTabs', () => {
 		expect( screen.queryByRole( 'tab', { name: 'Share', selected: false } ) ).toBeVisible();
 		expect( screen.queryByRole( 'tab', { name: 'Settings', selected: false } ) ).toBeVisible();
 		expect( screen.queryByRole( 'tab', { name: 'Assistant', selected: false } ) ).toBeNull();
+		expect( screen.queryByRole( 'tab', { name: 'Backup', selected: false } ) ).toBeNull();
```

Comment:
Can we call the feature ImportExport and use `Import / Export` as a label, as we have it in the design and P2 discussion?

Reply:
Done! Thanks @wojtekn  for your comment and the clarification. I also updated the naming to be less confusing